models/basgra/Dockerfile

```diff
@@ -0,0 +1,20 @@
+# this needs to be at the top, what version are we building
+ARG IMAGE_VERSION="latest"
+
+# ----------------------------------------------------------------------
+# BUILD PECAN FOR BASGRA
+# ----------------------------------------------------------------------
+FROM pecan/models:${IMAGE_VERSION}
+
+# ----------------------------------------------------------------------
+# SETUP FOR SPECIFIC BASGRA VERSION
+# ----------------------------------------------------------------------
+
+# Some variables that can be used to set control the docker build
+ARG MODEL_VERSION=BASGRA_N_v1.0
+
+# Setup model_info file
+COPY model_info.json /work/model.json
+RUN sed -i -e "s/@VERSION@/${MODEL_VERSION}/g" \
+           -e "s#@BINARY@#/usr/local/lib/R/site-library/PEcAn.BASGRA/libs/PEcAn.BASGRA.so#g" /work/model.json
```

Comment:
probably want to figure out a better way to do the binaries for models that are in R.